scripts/zones/Arrapago_Remnants/npcs/Socket.lua

```diff
@@ -0,0 +1,31 @@
+-----------------------------------
+-- Area: Arrapago Remnants
+-- NPC: Socket
+-- Trade Cells to pop NM
+-----------------------------------
+local ID = require("scripts/zones/Arrapago_Remnants/IDs")
+-----------------------------------
+
+function onTrade(player,npc,trade)
+    local instance = npc:getInstance()
+    local mob = instance:getEntity(bit.band(ID.mob[2][3].wahzil, 0xFFF), dsp.objType.MOB)
+    local COUNT = trade:getItemCount()
+        
+    for i = 5365, 5384 do
```

Comment:
what are these numbers

Reply:
The cells you trade to the slot to pop a nm which in then turns upon defeat will award 2x the cells traded

Reply:
ah yea right, missed the item check below
could you comment in the start itemid name anyways